projects/plugins/jetpack/class.jetpack.php

```diff
@@ -4158,13 +4158,8 @@ public function deactivate_dialog( $hook ) {
 					true
 				);
 
-				wp_register_script(
-					'jp-tracks-functions',
-					plugins_url( '_inc/lib/tracks/tracks-callables.js', JETPACK__PLUGIN_FILE ),
-					array( 'jp-tracks' ),
-					JETPACK__VERSION,
-					false
-				);
+				$tracking = new Tracking();
+				$tracking->register_tracks_scripts();
```

Comment:
Nicely done! We're making sure that `jp-tracks-functions` is available as a dependency for other scripts. Let's do the same thing in `search.php`?

Reply:
Done.